content/en/hugo-modules/use-modules.md

```diff
@@ -30,9 +30,22 @@ Use `hugo mod init` to initialize a new Hugo Module. If it fails to guess the mo
 hugo mod init github.com/gohugoio/myShortcodes
 ```
 
-
 Also see the [CLI Doc](/commands/hugo_mod_init/).
 
+## Use a Module for a Theme
+The easiest way to use a for a theme is to import it in the config. 
+
+1. Initialize the hugo module system: `hugo mod init github.com/<your_user>/<your_project>`
+2. Import the theme in your `config.toml`: 
+
+```toml
+[module]
+  [[module.imports]]
+    path = "github.com/dzello/reveal-hugo"
```

Comment:
Thanks. It would be great, though, if you could pick a "more neutral" theme as an example, e.g. https://github.com/spf13/hyde/ (ideally we should have a simple theme in the gohugoio org, but spf13 did start this project, so might as well give him some PR).